source/common/http/http1/codec_impl.cc

```diff
@@ -101,8 +101,10 @@ void StreamEncoderImpl::encodeHeaders(const HeaderMap& headers, bool end_stream)
                    Headers::get().TransferEncoding.get().size(),
                    Headers::get().TransferEncodingValues.Chunked.c_str(),
                    Headers::get().TransferEncodingValues.Chunked.size());
-      // We do not aply chunk encoding for HTTP upgrades. Any incoming chunks
-      // will be through-proxied by maybeDirectDispatch untouched.
+      // We do not aply chunk encoding for HTTP upgrades.
+      // If there is a body in a WebSocket Upgrade response, the chunks will be
+      // passed through via maybeDirectDispatch so we need to avoid appending
+      // extra chunk boundaries endEncode.
```

Comment:
I think the "endEncode" is errant here. Or do you mean "... boundaries in encodeData and endEncode." ?

Reply:
turns out we don't call encodeData (because we direct dispatch) but we do call endEncode (because reasons), but I think documenting that is more likely to cause confusion so I'll just remove it :-)